react/Menu/index.jsx

```diff
@@ -132,16 +162,91 @@ class Menu extends Component {
   }
 }
 
+class IternalMenuItem extends Component {
+  constructor() {
+    super()
+    this.element = ''
+    this.state = {
+      style: { display: 'block', visibility: 'hidden', position: 'fixed' }
+    }
+  }
+  //When the component is ready, than let's do some Maths
+  componentDidMount() {
+    this.handleDisplay()
+  }
+
+  handleDisplay() {
```

Comment:
I find handleDisplay not very telling, something like `computeCoordinates` would be clearer IMHO